src/cli.js

```diff
@@ -6,12 +6,16 @@ import figlet from 'figlet';
 import inquirer from 'inquirer';
 import isValid from 'is-valid-path';
 import fs from 'fs';
-import utils from './utils';
+import Preferences from 'preferences';
+import winston from 'winston';
 
+import utils from './utils';
 import ciChoices from './constants/ci-choices';
 import containerizationChoices from './constants/containerization-choices';
 import constants from './constants/constants';
 
+const preferences = new Preferences(constants.hammer.name);
```

Comment:
I believe this line will need to be modified with the branding update. `constants.hammer.name` doesn't exist anymore, I don't think.

Reply:
I'm currently going through and fixing merge conflicts.